site/docs/relations/logical_relations.md

```diff
@@ -268,14 +268,23 @@ The set operation encompasses several set-level operations that support combinin
+* m: the number of time a records occurs in the primary input (p)
+* n1: the number of times a record occurs in the 1st secondary input (s1)
+* n2: the number of times a record occurs in the 2nd secondary input (s2)
+* ...
+* n: the number of times a record occurs in the nth secondary input
+
+| Operation                   | Description                                                                                                                                                                                                                                                                | Examples                                                                                                                                                        | Output Nullability
+|-----------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------| -----------------------------
+| Minus (Primary)             | Returns all records from the primary input excluding any matching rows from secondary inputs, removing duplicates.<br/>Each value is treated as a unique member of the set, so duplicates in the first set don’t affect the result.<br/>This operation maps to SQL EXCEPT. | MINUS<br/>&nbsp;&nbsp;p: {1, 2, 2, 3, 3, 3, 4}<br/>&nbsp;&nbsp;s1: {1, 2}<br/>&nbsp;&nbsp;s2: {3}<br/>YIELDS<br/>{4}                                            | The same as the primary input.                                                                                                                                                                     
+| Minus (Primary All)         | Returns all records from the primary input excluding any matching records from secondary inputs.<br/>For each specific record returned, the output contains max(0, m - sum(n1, n2, …, n)) copies.<br/>This operation maps to SQL EXCEPT ALL.                               | MINUS ALL<br/>&nbsp;&nbsp;p: {1, 2, 2, 3, 3, 3, 3}<br/>&nbsp;&nbsp;s1: {1, 2, 3, 4}<br/>&nbsp;&nbsp;s2: {3}<br/>YIELDS<br/>{2, 3, 3}                            | The same as the primary input.
+| Minus (Multiset)            | Returns all records from the primary input excluding any records that are included in *all* secondary inputs.<br/>This operation does have a direct SQL mapping.                                                                                                           | MINUS MULTISET<br/>&nbsp;&nbsp;p: {1, 2, 3, 4}<br/>&nbsp;&nbsp;s1: {1, 2}<br/>&nbsp;&nbsp;s2: {1, 2, 3}<br/>YIELDS<br/>{3, 4}                                   | The same as the primary input.
```

Comment:
minor: I assume you mean "This operation does **not** have a direct SQL mapping."